lib/ignored-paths.js

```diff
@@ -188,4 +190,36 @@ IgnoredPaths.prototype.contains = function(filepath, category) {
 
 };
 
+/**
+ * Returns a list of dir patterns for glob to ignore
+ * @returns {Array} list of glob ignore patterns
+ */
+IgnoredPaths.prototype.ignoredFoldersGlobPatterns = function() {
```

Comment:
I'd call this `getIgnoredFoldersGlobPatterns` (our conventions have methods beginning with verbs)